web/pdf_print_service.js

```diff
@@ -310,7 +310,8 @@ if ('onbeforeprint' in window) {
 var overlayPromise;
 function ensureOverlay() {
   if (!overlayPromise) {
-    overlayPromise = OverlayManager.register('printServiceOverlay',
+    overlayManager = PDFViewerApplication.overlayManager;
```

Comment:
I have not been able to trigger that as well, but for safety I added this check in the new commit.